pkg/cmd/api/api.go

```diff
@@ -298,12 +301,22 @@ func fillPlaceholders(value string, opts *ApiOptions) (string, error) {
 		return value, err
 	}
 
+	var branch string
+	if strings.Contains(value, ":branch") {
```

Comment:
There's a subtle bug here: the `placeholderRE` matches only `:branch\b` (notice the word boundary), but `strings.Contains` matches ":branch" even when it's not on a word boundary.

So I would move the branch lookup logic to the below `case ":branch"` block